common/diagnostics/MonitoringServer.java

```diff
@@ -114,7 +114,7 @@ public void channelRead0(ChannelHandlerContext ctx, HttpObject msg) {
                 FullHttpResponse response = new DefaultFullHttpResponse(req.protocolVersion(), OK);
                 String query = Objects.requireNonNullElse(uri.getQuery(), "");
                 if (query.toLowerCase().contains("format=json")) {
-                    response.content().writeBytes(Unpooled.wrappedBuffer((metrics.formatJSON()).getBytes(StandardCharsets.UTF_8)));
+                    response.content().writeBytes(Unpooled.wrappedBuffer((metrics.formatJSON(false)).getBytes(StandardCharsets.UTF_8)));
```

Comment:
Boolean flags need to die in a fire. What's a false JSON?

Reply:
Moving the conversation to `common/diagnostics/Metrics.java`

Reply:
I didn't want to create two methods `formatJSONMonitoring` and `formatJSONReporting` and wasn't sure what `typedb` thinks about boolean flags as arguments. I used to mark it this way in C++:
```
methodCall(
true, // flag1's name
false) // flag2's name
``` 
But everyone here (except just you?) use Intellij which shows the name of the argument inside the IDE, so I was hesitant about it. 

Based on the related question: `I thought the monitoring page was supposed to use the same data as Prometheus, with counters steadily ticking up?`, it could be a more clear difference between monitoring and reporting JSONs, so creating two methods would be better in terms of design than in the current state, so I'd just change it to two methods.